src/pages/Order/Order.js

```diff
@@ -1,8 +1,123 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { ORDER_INPUT_DATA } from './orderInputData';
 import './Order.scss';
 
 const Order = () => {
-  return <div>Order</div>;
+  const [inputValue, setInputValue] = useState({
+    userPhoneNumber: '',
+    userAddress: '',
+  });
+
+  const [orderProductData, setOrderProductData] = useState([]);
+
+  const userPhoneNumberCheck = inputValue.userPhoneNumber.length > 3;
+
+  const isShowHintText = _title =>
+    inputValue.userPhoneNumber &&
+    _title === '전화번호' &&
+    !userPhoneNumberCheck;
+
+  function handleValueChange(e) {
+    const { value, name } = e.target;
+    setInputValue({ ...inputValue, [name]: value });
+  }
+
+  const handleSubmit = () => {
+    fetch('', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json; charset=utf-8',
+      },
+      body: JSON.stringify(inputValue),
+    })
+      .then(response => response.json())
+      .then(data => data);
+  };
+
+  useEffect(() => {
+    fetch('/data/orderData.json')
+      .then(res => res.json())
+      .then(data => {
+        setOrderProductData([data]);
```

Comment:
```suggestion
        setOrderProductData(data);
```
배열에 꼭 안 감싸주어도 될 것 같습니다!